app/forms.py

```diff
@@ -1,4 +1,4 @@
-from flask_babel import lazy_gettext as _l
+from flask_babel import lazy_gettext as _l # ten as jest tu niepotrzebny i utrudnia czytanie kodu
```

Comment:
W każdym wystąpieniu "_l" zamienić na "lazy_gettext"? Czy to wtedy będzie bardziej czytelne? Więcej tekstu?

Reply:
Poprawione